components/filter/README.md

```diff
@@ -222,47 +222,54 @@ A tag-list allowing the user to see (and remove) the currently applied filters.
   import '@brightspace-ui/core/components/filter/filter-tags.js';
 </script>
 <style>
-	d2l-filter-tags {
-		max-width: calc(100% - 300px);
-	}
-
-	@media(max-width: 600px) {
-      d2l-filter-tags {
+  .filter-wrapper {
```

Comment:
Updated the example to use flex rather than a set max-width, which seems more ideal/like a potential usecase based on designs.